package.json

```diff
@@ -13,7 +13,7 @@
     "@fortawesome/fontawesome-svg-core": "^1.2.28",
     "@fortawesome/free-solid-svg-icons": "^5.13.0",
     "@fortawesome/vue-fontawesome": "^0.1.9",
-    "@scaife-viewer/scaife-widgets": "0.13.1",
+    "@scaife-viewer/scaife-widgets": "git://github.com/scaife-viewer/scaife-widgets#6fd5e8b8e2ce7f650cec0720efe19bb307d7f285",
```

Comment:
- [ ] Bump to release of `scaife-widgets` after geting feedback on https://github.com/scaife-viewer/scaife-widgets/pull/32